Add team page

jracusin opened the pull request:
<!-- IMPORTANT!!! Aside from typographical corrections and minor changes, please consider creating an Issue before making a Pull Request. -->

# Description
<!--  Please include a one to two sentence description that summarizes the issue and your solution. -->
Add team page listing current and former members of the GCN team.

# Related Issue(s)


# Testing
<!-- Describe how you tested this PR. Include step by step instructions for others to test this PR.
Be detailed and include images where appropriate. -->
local dev

lpsinger commented on app/routes/docs.team/route.tsx:
```diff
@@ -0,0 +1,101 @@
+}) {
+  return (
+    <>
+      <div className="usa-card__container margin-bottom-2 grid-col-4">
```
Use `<CardGroup>` here instead of a div.

lpsinger reviewed (changes requested):
- Add a link to this page.
- Add GitHub handles and links to GitHub profiles.
- Add headshots for each person. For me, you can use the image from my GitHub profile.

jracusin commented on app/routes/docs.team/route.tsx:
```diff
@@ -0,0 +1,101 @@
+        </TeamCard>
+        <TeamCard
+          name="Courey Elliott"
+          affiliation="Lousiana State University/ NextSource"
```
This space is there in order to get the right line break.

jracusin commented:
<img width="562" height="1118" alt="Screenshot 2025-09-02 at 2 11 49 PM" src="https://github.com/user-attachments/assets/b41fa443-06bd-4958-b34f-da54cf6323bc" />
<img width="447" height="1105" alt="Screenshot 2025-09-02 at 2 12 03 PM" src="https://github.com/user-attachments/assets/a0d2b5bf-d949-47ed-868b-1b7dcdf45ed4" />

lpsinger commented on app/routes/docs.team/route.tsx:
```diff
@@ -53,51 +42,34 @@ function TeamCard({
+          width="50"
+          height="50"
+          loading="lazy"
+          alt="headshot of GCN team member"
```
A "headshot" has the unfortunate second meaning of a gunshot to the head.

```suggestion
          alt="Photograph of GCN team member"
```

lpsinger commented on app/routes/docs.team/route.tsx:
```diff
@@ -53,51 +42,34 @@ function TeamCard({
+          src={photo}
+          width="50"
+          height="50"
+          loading="lazy"
```
Why lazy loaded?

```suggestion
```

lpsinger commented on app/routes/docs.team/route.tsx:
```diff
@@ -53,51 +42,34 @@ function TeamCard({
+      <CardMedia imageClass="bg-white margin-bottom-neg-2">
+        <img
+          src={photo}
+          width="50"
+          height="50"
```
Are these images actually square?

lpsinger commented on app/routes/docs.team/route.tsx:
```diff
@@ -53,51 +42,34 @@ function TeamCard({
-      )}
-    </>
+    <Card className="grid-col-fill">
+      <CardMedia imageClass="bg-white margin-bottom-neg-2">
```
Why the custom margin?

lpsinger reviewed (commented):
I will have some typographical suggestions on this, but I can't do that today because I need to finish my grant proposals.

However, I would suggest removing the group photos. If you want them, I would pick and keep just _one_.

Vidushi-GitHub commented:
> > I will have some typographical suggestions on this, but I can't do that today because I need to finish my grant proposals.
> > However, I would suggest removing the group photos. If you want them, I would pick and keep just _one_.
> 
> I like the Team photos, and it's difficult to keep just one because we don't have only one with everyone in it. I'm open to suggestions to make them less prominent. I could move them all to the end and/or make them smaller.

I think two photos, the one at NASA and other at MN are sufficient. Otherwise, it looks like an album gallery.

jracusin commented:
I removed team photos after discussion of their use elsewhere and rebased.  I also add another TeamCard for when we don't want to link to GitHub (e.g. past team members).

jracusin commented:
I replaced the href CSS with the Link component and added the GitHub handle as part of the TeamCard component. I'm not sure what other incline CSS can be replaced with USWDS components.

I asked Teresa for a photo, but haven't heard back yet.  I'd like her permission.
